### PR TITLE
Simulation of skipping ads

### PR DESCRIPTION
**Pull request by thanasistrisp**
After issue #197, I tried to implement skipping ads by going forward to 29.9th second (it is constant always) using playerctl. It seems to work well.
However, there are some issues that should be fixed (if @Mastermindzh could help):

- [ ] The command is based on shell using fork technique. This should use [MPRIS](https://www.npmjs.com/package/mpris-service) API (else playerctl should be installed explicitly to system) but I do not know exactly how to call it:  I do not see the position command to API.
- [ ] Playerctl command should be called with --player option. If I try to run through shell the command `playerctl --player=tidal-hifi position 29.9`, ~~I get the error `Could not execute command: Could not get track id to set position`. There something goes wrong. Track id is not set at all:~~ it does nothing yet.
- [x] For better user experience, it would be better to also mute music these milliseconds of skipping as sometimes (not always) at the beginning, the ad maybe heard for a little (~ <1 second).
- [x] Hide upgrade banner and other annoying connection messages. Network issues are kept untouched.

### CODE REVIEW

File: src/preload.js [hunk 1]
```diff
@@ -407,7 +407,21 @@ setInterval(function () {
     if (store.get(skipArtists)) {
       const skippedArtists = store.get(settings.skippedArtists);
       if (skippedArtists.find((artist) => artist === artists) !== undefined) {
-        elements.click("next");
+        if (artists === "TIDAL") {
```
Comment (Mastermindzh) [inline src/preload.js:410]: Please put this behind a feature flag in the settings & extract it to a function.
Other than that it looks good. 😄 


*Additional*

One boy scout fix that you could also make (but not required) in the rest of the code is bundling the 2 mute lines into a function:

```typescript
setMuted(muted: boolean){
   elements.click("volume");
   isMutedArtist = muted;
}
```
You can then use it in this part of the code but also in the rest of the code:

```typescript
          if (!elements.isMuted()) {
              setMuted(true);
          }
```

**Review by Mastermindzh — CHANGES_REQUESTED** [review]
Please put it behind a setting and extract the function 😄

**Comment by Mastermindzh** [comment]
Playerctl is not an issue i.m.o, mpris doesn't offer the control you need IIRC.
I don't know the answer to the second answer, I'm not sure I understand it yet either. Will have to look more closely but I don't have much time.

File: src/preload.js (hunk 1 above)
Comment (thanasistrisp) [reply src/preload.js]: Fixed it. See commit message for more details.

**Comment by thanasistrisp** [comment]
> Playerctl is not an issue i.m.o, mpris doesn't offer the control you need IIRC. I don't know the answer to the second answer, I'm not sure I understand it yet either. Will have to look more closely but I don't have much time.

Yes, I do not disagree. The problem is not the command line option, but the issue that `playerctl` command is called without `--player=tidal-hifi` option, which could cause issues if another player supported by mpris service is played simultaneously. 
To make it work, I need to set `"mpris:trackid"` option, but I do not fully understand what exactly is this here. If you have time, take a look.

**Comment by thanasistrisp** [comment]
Also, I removed muted Artists option as it is not needed anymore. It just adds workload to the loop without reason, and it causes conflicts with ad blocking. Now it is ok, less than one second, to skip the ad. I also set the interval time to 400ms so that options get updated more quickly.

**Comment by thanasistrisp** [comment]
Ready for merge for now. Sometime later, we will have to look for playerctl player option I said above to have more precise results.

**Comment by thanasistrisp** [comment]
Closed for real ad blocking in #218.

**Pull request closed without merging**